sdk/purview/azure-analytics-purview-workflow/swagger/README.md

```diff
@@ -2,7 +2,7 @@
 
 ```yaml
 input-file:
-- https://github.com/Azure/azure-rest-api-specs/blob/3a1ca5a2130250382e65e4585d8e88bfab6d93e4/specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/purviewWorkflow.json
+- https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2023-10-01-preview/purviewWorkflow.json
```

Comment:
Use a permalink (ctrl+shift+,) because the commit in the "main" could be different the next time someone runs this.

Reply:
Fixed thanks.